pillar/fluentd/consul.sls

```diff
@@ -15,24 +14,8 @@ fluentd:
             - pos_file: /var/log/syslog.pos
             - format: syslog
             - tag: consul.server
-        - directive: filter
-          directive_arg: '**'
-          attrs:
-            - '@type': grep
-            - regexp1: ident consul
         - {{ auth_log_source('syslog.auth', '/var/log/auth.log') }}
-        - {{ auth_log_filter('grep', 'ident', 'CRON') }}
+        - {{ auth_log_filter('grep', 'ident', 'consul', 'consul.server', 'regexp') }}
```

Comment:
I think we want to move this above the auth_log_source line so that we don't allow only Consul messages from the auth log as well.

Reply:
But the filter is using the `consul.server` tag which should only apply to the /var/log/syslog file and not auth.log one

Reply:
Fair point, missed that.